doc/rados/operations/crush-map-edits.rst

```diff
@@ -265,7 +265,7 @@ all of the
 chassis's hosts and their associated OSDs will be in the ``down``
 state and thus unavailable.  It is important to avoid placing multiple replicas
 or shards of data within a single such chassis, which in this case is
-a _failure domain_.
+a *failure domain*.
```

Comment:
I dunno what I was thinking that day.  Too much switching between Slack and RST.